pkg/services/grpcserver/interceptors/logging.go

```diff
@@ -17,11 +18,18 @@ func LoggingUnaryInterceptor(cfg *setting.Cfg, logger log.Logger) grpc.UnaryServ
 	) (resp any, err error) {
 		resp, err = handler(ctx, req)
 		if cfg.GRPCServerEnableLogging {
+			md, _ := metadata.FromIncomingContext(ctx)
+
+			// Redact grafana token from logs
```

Comment:
https://github.com/grafana/authlib/blob/main/authn/grpc_client_interceptors.go#L16

should also check for:
* "X-Access-Token"
* "X-Id-Token"

or maybe anything ending in token?